ERCs/ERC721/ERC721-TokenScript.xml

```diff
@@ -42,7 +42,7 @@
             <ts:origins>
                 <!-- this gives you card attributes from, to, tokenID, which is different than the
                  token's attributes -->
-                <ethereum:event type="Transfer" filter="${_from}=${ownerAddress}"/>
+                <ethereum:event type="Transfer" filter="${from}=${ownerAddress}"/>
```

Comment:
No interpolation on the LHS, so it should be:

```
filter="from=${ownerAddress}"
```

This should be fixed in master already.